index.html

```diff
@@ -8,7 +8,29 @@
     <title>Microverse</title>
 </head>
 <body>
-    <h1>Hello Microverse</h1>
+    <div class="nav-bar">
+        <div class="logo">Yash Solo</div>
+        <div class="ham-manu"><img src="res/ham-manu.svg" alt="ham-menu"></div>
+    </div>
+    <div class="hero-sec">
+        <div class="hero-main-header"><h1>Hey There.</br>
+            I’m Yash</h1></div>
+        <div class="hero-sm-header"><h3>I’am a Software Developer</h3></div>
+        <div class="hero-text"><p>I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+            don’t hestiate to contact me.</p></div>
+        <div class="social-media-sec">
+            <img class="social-ico" src="res/git.svg" alt="git">
+            <img class="social-ico" src="res/linkedin.svg" alt="linkedin">
+            <img class="social-ico" src="res/vic.svg" alt="victory">
+            <img class="social-ico" src="res/twitter.svg" alt="twitter">
+            <img class="social-ico" src="res/m.svg" alt="meta">
+        </div>
```

Comment:
- [ ]  Kyndly use a `ul` tag to  wrap the social media icons using an `li` tag per icon also you can consider using an `a` tag so you can referenciate the icons with your social media liks 

- Take this as an example 

![image](https://user-images.githubusercontent.com/95243996/191745748-ee20d9e7-5a7e-4826-ac97-726d760c3db5.png)